### DISCUSSION TITLE
Less than flag - TGOV and Exciter Models

### DISCUSSION THREAD

**Discussion by tyler477** [CURENT/andes#580] [Modeling]
May I double check some of the details in the documentation for the exciter/governor models? 

Is the coefficient of the 'less than' filter - which is Z1_LTLL - always equal to 1? What is its nominal value in (for instance) the TGOV1 model?

**Comment by cuihantao** [comment]
Its value should be evaluated on the fly. Z1 equals 1 means the “less than” evaluate to true. On Nov 18, 2024, at 10:00 PM, tyler477 ***@***.***> wrote:﻿
May I double check some of the details in the documentation for the exciter/governor models?
Is the coefficient of the 'less than' filter - which is Z1_LTLL - always equal to 1? What is its nominal value in (for instance) the TGOV1 model?

**Reply by tyler477** [reply]
Great. What is the purpose of {z1_LTLL}^2 then? is there a missing coefficient? or is the point of this to ensure a positive value? 

For example, in the TGOV1 equation for y_LL: 

y_LL=T_2 (-xprime_LL+y_LAG )+T_3 (xprime_LL-y_LL)+(z_1^(LT_LL ) )^2 (-xprime_LL+y_LL )

To write this logic back: 
	The flag is comparing xprime_LL and y_LL

	If the last timestep passes the flag, xprime_LL<y_LL, then (z_1^(LT_LL ) )^2=1

	Else (z_1^(LT_LL ) )^2=0 


Would this be right? The flag essentially determines whether (z_1^(LT_LL ) )^2 (-xprime_LL+y_LL ) is included in this step?
